client/main.lua

```diff
@@ -190,15 +190,22 @@ RegisterNetEvent('police:client:SendPoliceEmergencyAlert', function()
 end)
 
 -- Threads
+
 CreateThread(function()
+    -- Create markers for police stations
```

Comment:
```suggestion
    -- Create markers for police stations
    if not next(config.locations.stations) then return end
```

By doing this we don't have to worry about disabling blips instead we can just remove the ones from the configuration we don't need.